ext/random/random.c

```diff
@@ -86,7 +86,6 @@ PHPAPI uint32_t php_random_range32(php_random_algo_with_state engine, uint32_t u
 	uint32_t count = 0;
 
 	result = 0;
-	total_size = 0;
```

Comment:
Please do not remove this one for “visual consistency” with the loop at the bottom. Instead remove the assignment on declaration.

You could also reduce the scope of some of the variables by moving their declaration down (`count` is only used starting in line 114, `limit` only starting line 113).

Applies likewise to range64.

Reply:
That makes sense. I adjusted the code.